flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "only-arches": ["x86_64"],
```

Comment:
It seems MIT, is aarch64 not available? Can you build from source?

Reply:
I've not built from source yet because the building process is complicated (require multiple builds and package downloads using luarocks) + the base game is not available on aarch64, so I'm not sure if it's relevant

Reply:
it's still relevant do build from source. Even for a single architecture.